File: python/PatternPrinting/Alphabet_Pyramid_Mast.py
#x=int(input("enter a number:"))
#for i in range(1,x+1):
   # for j in range(1,i+1):
      #  print(chr(j),end=' ')
    # print()

x=int(input("enter a number:"))
a=65
for i in range(1,x+1):
    for j in range(1,i+1):
        print(chr(a),end=' ')
        a+=1
    print()

File: c/Pattern_Printing/Alphabet_Pyramid_mast.c
#include<stdio.h>
int main(){
    int n;
    printf("Enter a number:");
    scanf("%d",&n);
    for(int i=1;i<=n;i++){
        
        for(int j=1;j<=n-i;j++){
            printf("  ");
        }
        for(int k=1;k<=i;k++){
            char ch=(char)(k+64);
            printf("%c ",ch);
        }
        int a= i-1;
        for(int y=i-1;y>=1;y--){
        char ch=(char)(y+64);
            printf("%c ",ch);
            a--;
        }
        printf("\n");
    }
    return 0;
}


